src/components/Clipboard/Clipboard.tsx

```diff
@@ -50,12 +56,42 @@ export function Clipboard(props: ClipboardProps) {
     }, 3000);
   }
 
+  function handleMouseMove(event: React.MouseEvent<HTMLDivElement>) {
+    setPosition({ x: event.clientX, y: event.clientY });
+  }
+
+  const divStyle: CSSProperties = {
+    position: 'fixed',
+    left: position.x + 30,
+    top: position.y - 30,
+  };
+
+  function renderAddressHover() {
+    return (
+      <p
+        style={divStyle}
+        className="relative bg-secondary z-10 top-2 text-f-primary pr-2 w-fit truncate pointer-events-none px-1 rounded"
+      >
+        {rawContent}
+      </p>
+    );
+  }
```

Comment:
```suggestion
  function displayAddressOnHover() {
    const divStyle: CSSProperties = {
      position: 'fixed',
      left: position.x + 30,
      top: position.y - 30,
    };
  
    return (
      <div
        style={divStyle}
        className="relative bg-secondary z-10 top-2 text-f-primary pr-2 w-fit truncate pointer-events-none px-1 rounded"
      >
        {rawContent}
      </div>
    );
  }
```

If `divStyle` is only use in `displayAddressOnHover`, you can move this `const` into the function. 🤔

Reply:
thanks for that !

Reply:
@pivilartisant Can you rename the function as I wrote above? It's to make it more explicit 😉

Reply:
ouaip j'avais pas vu